docs/core/allocation.qmd

```diff
@@ -6,6 +6,34 @@ Allocation is the process of assigning an allocated abstraction flow rate to use
 
 The allocation problem is solved per subnetwork of the Ribasim model, from which a new graph is created called the allocation graph. The allocation graph is used to formulate an optimization problem with the [JuMP](https://jump.dev/JuMP.jl/stable/) package, which is solved using the [HiGHS solver](https://highs.dev/). For more information see also the example of solving the maximum flow problem with `JuMP.jl` [here](https://jump.dev/JuMP.jl/stable/tutorials/linear/network_flows/#The-max-flow-problem).
 
+```{mermaid}
```

Comment:
The Mermaid source indicates that this is a sequence diagram, but that term doesn't appear in the output. Could you briefly explain in the text above that this is a sequence diagram with a link to for instance https://en.wikipedia.org/wiki/Sequence_diagram? That could help for people that don't know how to read it, like me 5 minutes ago.